grawlix-cms/_admin/book.page-edit.php

```diff
@@ -776,8 +841,8 @@
 
 ?>
 
-
-<form accept-charset="UTF-8" action="book.page-edit.php" method="post" enctype="multipart/form-data">
+// Add name to the form for the script to find it
```

Comment:
This kind of comment makes sense for a commit message, but doesn't help the code be any more legible. Odd as it feels to say this, we can scrap it this comment :D